### ISSUE TITLE
using ambrose with hive 0.14

### ISSUE THREAD

**Issue by rjangid1** [twitter/ambrose#182]
I am trying to use ambrose on hortonworks sandbox and demo seemed to work perfectly fine . 
while trying to use with hive 0.14 I got this error 

```
hive.exec.driver.run.hooks Class not found:com.twitter.ambrose.hive.AmbroseHiveFinishHook
FAILED: Hive Internal Error: java.lang.ClassNotFoundException(com.twitter.ambrose.hive.AmbroseHiveFinishHook)
java.lang.ClassNotFoundException: com.twitter.ambrose.hive.AmbroseHiveFinishHook
	at java.net.URLClassLoader$1.run(URLClassLoader.java:366)
	at java.net.URLClassLoader$1.run(URLClassLoader.java:355)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.net.URLClassLoader.findClass(URLClassLoader.java:354)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:425)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:358)
	at java.lang.Class.forName0(Native Method)
	at java.lang.Class.forName(Class.java:274)
	at org.apache.hadoop.hive.ql.hooks.HookUtils.getHooks(HookUtils.java:60)
	at org.apache.hadoop.hive.ql.Driver.getHooks(Driver.java:1267)
	at org.apache.hadoop.hive.ql.Driver.runInternal(Driver.java:1096)
	at org.apache.hadoop.hive.ql.Driver.run(Driver.java:1006)
	at org.apache.hadoop.hive.ql.Driver.run(Driver.java:996)
	at org.apache.hadoop.hive.cli.CliDriver.processLocalCmd(CliDriver.java:247)
	at org.apache.hadoop.hive.cli.CliDriver.processCmd(CliDriver.java:199)
	at org.apache.hadoop.hive.cli.CliDriver.processLine(CliDriver.java:410)
	at org.apache.hadoop.hive.cli.CliDriver.processLine(CliDriver.java:345)
	at org.apache.hadoop.hive.cli.CliDriver.processReader(CliDriver.java:443)
	at org.apache.hadoop.hive.cli.CliDriver.processFile(CliDriver.java:459)
	at org.apache.hadoop.hive.cli.CliDriver.executeDriver(CliDriver.java:739)
	at org.apache.hadoop.hive.cli.CliDriver.run(CliDriver.java:677)
	at org.apache.hadoop.hive.cli.CliDriver.main(CliDriver.java:616)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.apache.hadoop.util.RunJar.run(RunJar.java:221)
	at org.apache.hadoop.util.RunJar.main(RunJar.java:136)
```

any suggestion for how to get it working ?

Thanks

**Comment by billonahill** [comment]
Make sure the jar that contains that class is in the classpath of the hive job you're trying to launch.

**Comment by rjangid1** [comment]
Hi @billonahill I tried moving the jar in path and hive job started as intended. but after finishing it showed a different error which seems to be related to hive 0.14 only. 

I checked hive 0.14's source code and couldn't find this method getLastMapRedStatsList. 

any plans to upgrade ambrose to include hive 0.14 ?

![screen shot 2016-05-18 at 1 15 27 pm](https://cloud.githubusercontent.com/assets/10449098/15351215/d3fce058-1cfb-11e6-9f30-d18bbdeb41a2.png)
